packages/ui/src/Layout/components/PluginItem.tsx

```diff
@@ -0,0 +1,183 @@
+    display: "block",
+    textDecoration: "none",
+    color:
+      theme.colorScheme === "dark"
+        ? theme.colors.dark[0]
+        : theme.colors.gray[7],
+    padding: `0 ${theme.spacing.md}px`,
+    fontSize: theme.fontSizes.sm,
+    marginRight: theme.spacing.md,
+    fontWeight: 500,
+    height: 35,
+    lineHeight: "35px",
+
+    "&:hover": {
+      backgroundColor:
+        theme.colorScheme === "dark"
+          ? theme.colors.dark[5]
+          : theme.colors.gray[1],
+      color: theme.colorScheme === "dark" ? theme.white : theme.black
+    }
+  },
+
+  linkActive: {
+    "&, &:hover": {
+      borderLeftColor: theme.fn.variant({
+        variant: "filled",
+        color: theme.primaryColor
+      }).background,
+      backgroundColor: theme.fn.variant({
+        variant: "filled",
+        color: theme.primaryColor
+      }).background,
+      color: theme.white
+    }
+  },
+  hash: {
+    color: "black",
+    marginRight: 8
+  },
+  hashActive: { color: "white" }
+}));
+
+interface PluginItemProps {
+  icon?: TablerIcon;
+  label: string;
+  initiallyOpened?: boolean;
+  links?: { label: string; link: string; image?: string }[];
+}
+
+const PluginItem: React.FC<PluginItemProps> = ({
+  icon: Icon,
+  label,
+  links
+}) => {
+  const [active, setActive] = useState("");
+  const { classes, cx } = useStyles();
+  const hasLinks = Array.isArray(links);
+  const items = (hasLinks ? links : []).map(link => (
+    <Text<"a">
+      component="a"
+      className={cx(classes.link, {
+        [classes.linkActive]: active === link.label
+      })}
+      href={link.link}
+      key={link.label}
+      pl="40px"
+      onClick={event => {
+        event.preventDefault();
+        setActive(link.label);
+      }}
+    >
+      {link.image ? (
+        <Flex align={"center"}>
+          <Indicator
+            mr={4}
+            dot
+            inline
+            size={10}
+            offset={4.5}
+            position="bottom-end"
+            color="green"
+            withBorder
+          >
+            <Image
+              mr={5}
+              height={25}
+              width={25}
+              radius="xs"
+              withPlaceholder
+              src={link.image}
+              alt="Random unsplash image"
+            />
+          </Indicator>{" "}
+          {link.label}
+        </Flex>
+      ) : (
+        <Flex align={"center"}>
+          <IconHash
+            size={13}
+            className={cx(classes.hash, {
+              [classes.hashActive]: active === link.label
+            })}
+          />
+          {link.label}
+        </Flex>
+      )}
+    </Text>
+  ));
+  return (
+    <Accordion
+      p={0}
+      variant="contained"
+      chevronPosition="left"
+      chevron={<IconChevronRight size={12} />}
+      classNames={classes}
+      styles={{
+        chevron: {
+          "&[data-rotate]": {
+            transform: "rotate(90deg)"
+          }, marginRight:"3px"
+        }
+      }}
+    >
```

Comment:
the chevron should not be necessary since accordion has it by default

Reply:
The original icon is positioned in a different way😅. I'll try to use styles to deal with that then

Reply:
sounds good to me